tests/integration/test_charm.py

```diff
@@ -66,6 +66,29 @@ async def test_synapse_validate_configuration(synapse_app: Application):
     )
 
 
+async def test_enable_stats_exporter(
+    synapse_app: Application,
+    synapse_app_name: str,
+    get_unit_ips: typing.Callable[[str], typing.Awaitable[tuple[str, ...]]],
+) -> None:
+    """
+    arrange: Synapse is integrated with Postgresql.
+    act:  request Synapse Stats Exporter URL.
+    assert: the Synapse application is active and the Exporter returns as expected.
```

Comment:
The statement "the Synapse application is active" is not asserted after the act phase, am I right? This is a bit confusing.

Reply:
Good catch, changed.